[2023-09-01T16:01:50.331+0000] {processor.py:157} INFO - Started process (PID=33) to work on /opt/airflow/dags/s3_upload.py
[2023-09-01T16:01:50.332+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/s3_upload.py for tasks to queue
[2023-09-01T16:01:50.335+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:01:50.335+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:01:50.745+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:01:50.743+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/s3_upload.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/s3_upload.py", line 5, in <module>
    from extract_functions import get_monthly_games, get_blitz_game_statistics, get_rapid_game_statistics
ModuleNotFoundError: No module named 'extract_functions'
[2023-09-01T16:01:50.747+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:01:50.766+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/s3_upload.py took 0.439 seconds
[2023-09-01T16:02:21.039+0000] {processor.py:157} INFO - Started process (PID=41) to work on /opt/airflow/dags/s3_upload.py
[2023-09-01T16:02:21.041+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/s3_upload.py for tasks to queue
[2023-09-01T16:02:21.045+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:02:21.045+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:02:21.238+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:02:21.236+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/s3_upload.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/s3_upload.py", line 5, in <module>
    from extract_functions import get_monthly_games, get_blitz_game_statistics, get_rapid_game_statistics
ModuleNotFoundError: No module named 'extract_functions'
[2023-09-01T16:02:21.239+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:02:21.250+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/s3_upload.py took 0.216 seconds
[2023-09-01T16:02:51.315+0000] {processor.py:157} INFO - Started process (PID=49) to work on /opt/airflow/dags/s3_upload.py
[2023-09-01T16:02:51.318+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/s3_upload.py for tasks to queue
[2023-09-01T16:02:51.323+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:02:51.322+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:02:51.493+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:02:51.491+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/s3_upload.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/s3_upload.py", line 5, in <module>
    from extract_functions import get_monthly_games, get_blitz_game_statistics, get_rapid_game_statistics
ModuleNotFoundError: No module named 'extract_functions'
[2023-09-01T16:02:51.493+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:02:51.504+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/s3_upload.py took 0.194 seconds
[2023-09-01T16:03:21.781+0000] {processor.py:157} INFO - Started process (PID=57) to work on /opt/airflow/dags/s3_upload.py
[2023-09-01T16:03:21.783+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/s3_upload.py for tasks to queue
[2023-09-01T16:03:21.786+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:03:21.785+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:03:21.880+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:03:21.878+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/s3_upload.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/s3_upload.py", line 5, in <module>
    from extract_functions import get_monthly_games, get_blitz_game_statistics, get_rapid_game_statistics
ModuleNotFoundError: No module named 'extract_functions'
[2023-09-01T16:03:21.880+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:03:21.893+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/s3_upload.py took 0.116 seconds
[2023-09-01T16:03:52.207+0000] {processor.py:157} INFO - Started process (PID=65) to work on /opt/airflow/dags/s3_upload.py
[2023-09-01T16:03:52.209+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/s3_upload.py for tasks to queue
[2023-09-01T16:03:52.213+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:03:52.213+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:03:52.320+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:03:52.318+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/s3_upload.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/s3_upload.py", line 5, in <module>
    from extract_functions import get_monthly_games, get_blitz_game_statistics, get_rapid_game_statistics
ModuleNotFoundError: No module named 'extract_functions'
[2023-09-01T16:03:52.320+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:03:52.333+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/s3_upload.py took 0.131 seconds
[2023-09-01T16:04:22.533+0000] {processor.py:157} INFO - Started process (PID=73) to work on /opt/airflow/dags/s3_upload.py
[2023-09-01T16:04:22.536+0000] {processor.py:829} INFO - Processing file /opt/airflow/dags/s3_upload.py for tasks to queue
[2023-09-01T16:04:22.541+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:04:22.540+0000] {dagbag.py:539} INFO - Filling up the DagBag from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:04:22.648+0000] {logging_mixin.py:151} INFO - [2023-09-01T16:04:22.646+0000] {dagbag.py:347} ERROR - Failed to import: /opt/airflow/dags/s3_upload.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 343, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/s3_upload.py", line 5, in <module>
    from extract_functions import get_monthly_games, get_blitz_game_statistics, get_rapid_game_statistics
ModuleNotFoundError: No module named 'extract_functions'
[2023-09-01T16:04:22.648+0000] {processor.py:841} WARNING - No viable dags retrieved from /opt/airflow/dags/s3_upload.py
[2023-09-01T16:04:22.661+0000] {processor.py:179} INFO - Processing /opt/airflow/dags/s3_upload.py took 0.133 seconds
